Sergei_Kulina/06.Ansible.run/06.Ansible.run.md

```diff
@@ -0,0 +1,181 @@
+ok: [bridge_sa] => {
+    "msg": " OS: CentOS Realese: 7.7 "
+}
+ok: [host9] => {
+    "msg": " OS: CentOS Realese: 7.9 "
+}
+ok: [host10] => {
+    "msg": " OS: Ubuntu Realese: 18.04 "
+}
+
+TASK [Mount point] *************************************************************
+ok: [bridge_sa] => {
+    "msg": " Mount point: / Capacity: 10015.26953125 Mb Used: 2083.29296875 Mb "
+}
+ok: [host9] => {
+    "msg": " Mount point: / Capacity: 20030.578125 Mb Used: 1765.21875 Mb "
+}
+ok: [host10] => {
+    "msg": " Mount point: / Capacity: 20030.578125 Mb Used: 2270.16015625 Mb "
+}
+
+TASK [debug] *******************************************************************
+ok: [bridge_sa] => {
+    "msg": " RAM total mb: 4096 RAM free mb: 2739 "
+}
+ok: [host9] => {
+    "msg": " RAM total mb: 4096 RAM free mb: 3564 "
+}
+ok: [host10] => {
+    "msg": " RAM total mb: 4096 RAM free mb: 3198 "
+}
+
+```
+
+## User add and upgrade system
+```bash 
+ansible-playbook useradd.yaml -i inv.yaml -e "user_to_add=test"
+[WARNING]: Skipping callback plugin 'profile_task', unable to load
+[WARNING]: ansible.utils.display.initialize_locale has not been called, this may
+result in incorrectly calculated text widths that can cause Display to print incorrect
+line lengths
+
+PLAY [Managed by ansible] *************************************************************
+
+TASK [Gathering Facts] ****************************************************************
+ok: [host9]
+fatal: [k8s9]: UNREACHABLE! => {"changed": false, "msg": "Failed to connect to the host via ssh: ssh_exchange_identification: Connection closed by remote host", "unreachable": true}
+[WARNING]: Failure using method (v2_runner_on_unreachable) in callback plugin
+(<ansible_collections.community.general.plugins.callback.mail.CallbackModule object at
+0x7f908bec4e48>): 'in <string>' requires string as left operand, not int
+ok: [host10]
+fatal: [k8s10]: UNREACHABLE! => {"changed": false, "msg": "Failed to connect to the host via ssh: ssh_exchange_identification: Connection closed by remote host", "unreachable": true}
+fatal: [ansi5]: UNREACHABLE! => {"changed": false, "msg": "Failed to connect to the host via ssh: ssh_exchange_identification: Connection closed by remote host", "unreachable": true}
+
+TASK [Creating user  test] ************************************************************
+ok: [host9]
+changed: [host10]
+
+TASK [Set authorized key taken from file] *********************************************
+ok: [host10]
+ok: [host9]
+
+TASK [Check user] *********************************************************************
+changed: [host9]
+changed: [host10]
+
+TASK [debug] **************************************************************************
+ok: [host9] => {
+    "msg": [
+        "test:x:1001:100:User for test:/home/test:/bin/bash",
+        "Пн окт  4 20:40:17 UTC 2021"
+    ]
+}
+ok: [host10] => {
+    "msg": [
+        "t:x:1001:100:User for test:/home/t:/bin/sh",
```

Comment:
@Kulinych 
Some strange in name of user, could you please check if task was done properly